packages/sign-client/src/controllers/engine.ts

```diff
@@ -834,14 +853,9 @@ export class Engine extends IEngine {
     }
 
     await this.setProposal(fallbackId, { id: fallbackId, ...proposal });
-
-    await this.client.auth.requests.set(id, {
-      authPayload: request.authPayload,
-      requester: request.requester,
-      expiryTimestamp,
-      id,
+    await this.setAuthRequest(id, {
+      request: { ...request, verifyContext: {} as any },
```

Comment:
I know it's not new but why do we need an `any` cast here? Can we avoid it?

Reply:
It's cast to any since there is no verify attestation. I can fill it with dummy data instead of empty {}